### PR TITLE
chore(ci): checks the PR title according to semantic rel(#31)

### PR DESCRIPTION
**Pull request by git-devansh**

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -124,3 +130,9 @@ jobs:
       run: |
         pre-commit install
         makim tests.lint
+
```
Comment (xmnlab) [inline .github/workflows/main.yaml:133]: You don't need these changes here. Just keep the implementation in the job above, please. Thanks!

**Review by xmnlab — CHANGES_REQUESTED** [review]

**Comment by git-devansh** [comment]
Thanks for your comment. This is correct?

**Comment by xmnlab** [comment]
That looks better. It is missing 2 things:

- ensure you have pre-commit installed and  install the pre-commit hooks locally: `pre-commit install`
- change your PR title to fit the standard used by semantic release. You can check the standard defined here: https://github.com/arxlang/arx/blob/main/CONTRIBUTING.md#commit-message-format ..  probably something starting with `chore(ci)` would be enough

**Review by xmnlab — APPROVED** [review]
thanks for working on that @git-devansh !

sorry for the delay here. also feel free to ping me here on github or on discord to request more reviews!

**Pull request merged**